src/components/app-field.js

```diff
@@ -1,64 +1,157 @@
-import React from 'react'
-import { connect } from 'react-redux'
-import { Table, Icon, Popup } from 'semantic-ui-react'
-
-import main from '../css/app-field.css'
+import React from 'react';
+import { connect } from 'react-redux';
```

Comment:
Group them as well.
```js
import React from 'react'
import { connect } from 'react-redux'
import Container from 'semantic-ui-react'
import { isBrowser } from 'react-device-detect'

import CustomBreadcrumb from 'core/common/src/components/custom-breadcrumb'
import { getTheme } from 'formula_one'

import RedirectURIs from './redirect-uris'
import AppField from './app-field'

import main from '../css/app-detail.css'
```

1. Third party libraries (eg. react)
2. Components of other apps in omniport ecosystem (eg. formula_one)
3. Components in this app (developer)
4. CSS of this app